.github/workflows/links-fail-fast.yml

```diff
@@ -25,4 +25,4 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         with:
           fail: true
-          args: --max-concurrency 5 --cache --max-cache-age 1d .
+          args: --max-concurrency 5 --cache --max-cache-age 1d --accept 100..=103,200..=299,429 .
```

Comment:
can you add a TODO to remove the accept arg, since this is temporary?

Reply:
Just want to make sure we don't forget to revert this.

Reply:
And, add this issue link to the TODO comment: https://github.com/open-telemetry/opentelemetry-go-contrib/issues/6183